opr-example-server/README.md

```diff
@@ -1,9 +1,49 @@
 # OPR Example Server
 
 A very simple example server that shows what needs to be wired up to have
-a working OPR node. To start it, run:
+a working OPR node.
+
+## Getting it Running
+
+### With Docker
+To run the example server in Docker, you'll need to build the image and then run
+it locally.
+
+```
```

Comment:
I think I read the best practice is always to specify a language after the triple quotes in a code section (I learned that after I last edited the README, which is why it isn't like that everywhere). I believe the language to use in cases like this is "console":

Reply:
Done